app/helpers/form_helper.rb

```diff
@@ -220,7 +220,7 @@ def byte_size_f(f, attr, options = {})
     options[:label_help] = _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified.")
     options[:help_block] ||= soft_limit_warning_block
     options[:help_block] += f.hidden_field(attr, :class => "real-hidden-value", :id => nil)
-    options[:label_help_options] = { :rel => 'popover-modal' }
+    options[:label_help_options] = { :data => { :placement => 'right' } }
```

Comment:
Maybe we can just delete it, to be consistent with other fields whatever we decide to use globaly? Or is there a reason to be always on the right?

Reply:
It's just that other tooltips in the form are on the right. But it is a default, so I deleted it as suggested.